spec/Section 3 -- Type System.md

```diff
@@ -1692,6 +1692,13 @@ While defining a directive, it must not reference itself directly or indirectly:
 directive @invalidExample(arg: String @invalidExample) on ARGUMENT_DEFINITION
 ```
 
+When returning the set of directives from the `__Schema` introspection type, both
+@skip and @include directives must be included. @deprecated directive may be included
+if GraphQL implementation supports the type system definition language. 
+
+When representing a GraphQL schema using the type system definition language, both @skip,
+@include and @deprecated directives should be omitted for brevity.
```

Comment:
```suggestion
@include and @deprecated directives could be omitted for brevity.
```
We need that to be able to represent full introspection as SDL document, see: 
https://github.com/graphql/graphql-js/blob/c0cf320d3a569c83166ac93d36b2dfb81ebb5b75/src/utilities/__tests__/schemaPrinter-test.js#L593-L599

Reply:
Ok, at first I wrote `can`. Then hesitated, changed to `should`.